src/main/java/com/example/application/ui/horizontal/community/IdeasManagementView.java

```diff
@@ -1,28 +1,126 @@
 package com.example.application.ui.horizontal.community;
 
+import com.example.application.backend.entities.IdeaEntity;
 import com.example.application.ui.MainView;
 import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.dialog.Dialog;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.grid.GridVariant;
 import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
 import com.vaadin.flow.router.PageTitle;
 import com.vaadin.flow.router.Route;
 
+import java.util.Arrays;
+import java.util.List;
+
 /**
  *  IdeaManagement View shows ...
  *
- *  @author
+ *  @author Litharshi Sivarasa
  *  @version 1.0
  *  @since 15.12.2020
- *  @lastUpdated
+ *  @lastUpdated 18.01.2021
  */
 @Route(value = "ideasManagement", layout = MainView.class)
 @PageTitle("Ideenmanagement")
 public class IdeasManagementView extends Div {
 
+    private IdeaEntity ideaEntity = new IdeaEntity(1," "," "," ");
+
     public IdeasManagementView() {
         setId("ideasManagement");
         setClassName("pageContentPosition");
         addClassName("communityColorscheme");
-        add(new Text("Ideenmanagement"));
+
+        H2 ideaH = new H2("Ideenmanagement");
```

Comment:
Bitte diese H2 in eine H1 umwandeln, so dass sie mit allen anderen Überschriften aller Seiten angesprochen werden kann